docusaurus/docs/operate/quickstart/validator_cheatsheet.md

```diff
@@ -123,43 +178,36 @@ poktrolld query staking validator $VALIDATOR_ADDR
 
 This will display information about your validator, including its status and delegation.
 
-## Additional Commands
+## Validator FAQ
 
-- **Delegate additional tokens to your validator**:
+### How do I delegate additional tokens to my validator?
 
-  ```bash
-  poktrolld tx staking delegate $VALIDATOR_ADDR 1000000upokt --from your_account --chain-id=pocket-beta --gas=auto --gas-adjustment=1.5 --gas-prices=1upokt
-  ```
+```bash
+poktrolld tx staking delegate $VALIDATOR_ADDR 1000000upokt --from your_account --chain-id=pocket-beta --gas=auto --gas-adjustment=1.5 --gas-prices=1upokt
+```
 
-- **Unbond (undelegate) tokens from your validator**:
+### How do I unbond (undelegate) tokens from my validator?
 
-  ```bash
-  poktrolld tx staking unbond $VALIDATOR_ADDR 500000upokt --from your_account --chain-id=pocket-beta --gas=auto --gas-adjustment=1.5 --gas-prices=1upokt
-  ```
+```bash
+poktrolld tx staking unbond $VALIDATOR_ADDR 500000upokt --from your_account --chain-id=pocket-beta --gas=auto --gas-adjustment=1.5 --gas-prices=1upokt
+```
 
-- **Withdraw rewards**:
+### How do I withdraw my validator rewards?
 
-  ```bash
-  poktrolld tx distribution withdraw-rewards $VALIDATOR_ADDR --commission --from validator --chain-id=pocket-beta --gas=auto --gas-adjustment=1.5 --gas-prices=1upokt
-  ```
+```bash
+poktrolld tx distribution withdraw-rewards $VALIDATOR_ADDR --commission --from validator --chain-id=pocket-beta --gas=auto --gas-adjustment=1.5 --gas-prices=1upokt
+```
 
-- **Check validator's commission and rewards**:
+### How do I check my validator's commission and rewards?
```

Comment:
How do you feel if we remove this completely? I don't believe we mint tokens for validators. Here are mint module params:
```yaml
        inflation_rate_change: "0.0"
        inflation_max: "0.0"
        inflation_min: "0.0"
```

Reply:
Yea, let's remove it